recipes/vulkan-memory-allocator/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from conans import ConanFile, tools
+
+
+class VulkanMemoryAllocatorConan(ConanFile):
+    name = "vulkan-memory-allocator"
+    license = "MIT"
+    homepage = "https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Easy to integrate Vulkan memory allocation library."
+    topics = ("vulkan", "memory-allocator", "graphics")
+    requires = ("vulkan-headers/[>=1.0]")
+    no_copy_source = True
+    # No settings/options are necessary, this is header only
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        url = self.conan_data["sources"][self.version]["url"]
+        version = os.path.basename(url).replace(".zip", "")
+        if version.startswith('v'):
+            version = version[1:]
+        extracted_dir = "VulkanMemoryAllocator-" + version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def build(self):
+        # This is a header only
+        pass
```

Comment:
```suggestion
```

Reply:
Although a header-only library does not need a build step, the absence of it causes a warning during the `conan create` command. I'm not sure if it should be removed from the recipe.

Reply:
That's because you are missing the header_only attribut, see this comment below https://github.com/conan-io/conan-center-index/pull/3208#discussion_r506182111

Reply:
Actually it displays a warning even with `self.info.header_only()`.

@b1ackviking CCI recipes usually (always?) remove the build() method for header_only libs. I don't have a strong opinion weither `build()` should be kept or not to avoid this warning.
Anyway it's just to have consistent recipes.

Reply:
@prince-chrismc I checked that it still displays a warning.

@SpaceIm I could not find any header-only package that has a build() method. I agree that consistency with other recipes is a totally valid reason to remove it despite the produced warning.

Thank you both for the clarification on that.